tr/about/index.md

```diff
@@ -158,8 +158,8 @@ külfetini ortadan kaldırmıştır.
 Ruby çok çeşitli özelliklere sahiptir, aşağıda bir kaçından
 bahsedilmiştir:
 
-* Ruby Java ya da Python gibi istisna yakalama mekanizmalarına sahiptir,
-  hatalarla başetmek kolaylaşır.
+* Ruby; Java ya da Python gibi istisna yakalama mekanizmalarına sahiptir,
```

Comment:
Looking at the `en` version, `;` seems somewhat surprising, but I do not know anything about turkish punctuation...

Reply:
@sadikkuzu is the semicolon used here correct in Turkish? 😊